docs/05_best-practices/04_data-design-and-migration.md

```diff
@@ -2,9 +2,9 @@
 content_title: Data design and migration
 ---
 
-EOSIO based blockchains allow developers to easily update their smart contract code. However, a few things need to be considered when it comes to data update and/or migration. The main structure for storing data in EOSIO based blockchains is the multi index table. Once a multi index table has been created with a first version of a smart contract, it has some limitations when it comes to changing its structure. Below you will find a few possible approaches which you can consider when you design your smart contract data and its migration. 
+EOSIO based blockchains allow developers to easily update their smart contract code. However, a few things need to be considered when it comes to data update and/or migration. The main structure for storing data in EOSIO based blockchains is the multi-index table. Once a multi-index table has been created with a first version of a smart contract, it has some limitations when it comes to changing its structure. Below you will find a few possible approaches which you can consider when you design your smart contract data and its migration. 
 
-# How to modify the structure of a multi index table
+# How to modify the structure of a multi-index table
```

Comment:
If you agree MIT is an API - this should become something like 

How to modify a data structure defined using MIT